lib/actions.js

```diff
@@ -23,6 +23,37 @@ module.exports = function (config) {
       return reqInstance('PUT', '/users/me', body);
     },
 
+    // MASTER USERS
+    getUserByUUid: function (uuid) {
+      return reqInstance('GET', '/users/me/user/' + uuid)
+    },
+
+    getUsers: function (uuid) {
+      return reqInstance('GET', '/users/me/users/')
```

Comment:
@txssseal Is `uuid` in the args just a copy-paste mistake? The prefix `/users/me/...` seems like it might be just `/users` or `'/users/' + uuid`. I'm just poking around, please disregard if this code is stale.